projects/qgrid-core/sort/sort.state.d.ts

```diff
@@ -3,8 +3,9 @@
  *
  * * `'multiple'` allows to sort by several column keys.
  * * `'single'` allows to sort only by one column key.
+ * * `'explicit'` allows to sort multiple columns while Shift key pressed otherwise applies single mode
  */
-export declare type SortStateMode = 'single' | 'multiple';
+export declare type SortStateMode = 'single' | 'multiple' | 'explicit';
```

Comment:
what other variants do you have for the name?

Reply:
also please make this mode by default and make the `trigger` property empty.

Reply:
what about 'multipleKey' or 'keyMultiple'?
Also it could be 'mixed'

Reply:
yes great let's rename it to mixed.